src/blocks/share/inspector.js

```diff
@@ -61,10 +64,15 @@ class Inspector extends Component {
 		];
 
 		const defaultColors = [
+			{
+				value: blockBackgroundColor.color,
+				onChange: setBlockBackgroundColor,
+				label: __( 'Block Background Color' ),
+			},
 			{
 				value: backgroundColor.color,
 				onChange: setBackgroundColor,
-				label: __( 'Background Color' ),
+				label: __( 'Button Background Color' ),
```

Comment:
This should be "Button Color" and "Text Color" should change depending on which style is selected: ("Icon Color", "Text Color", "Icon and Text Color")

Reply:
Adjusted share to use dynamic labels depending on icon style. Also removed contrast checker for block background.